README.md

```diff
@@ -73,15 +73,33 @@ ssh:
 
 `private-key`: The path to the private key file to use for SSH authentication.
 
-`connect-timeout`: Maximum amount of time to allow for an SSH connection to be established, in seconds
+`connect-timeout`: Maximum amount of time to allow for an SSH connection to be established, in seconds.
+
+`tmpdir`: The directory to store temporary files on the target node. (default: location used by `mktemp -d`, usually `/tmp`)
+
+`run-as`: Triggers privilege escalation for commands on the target node as the specified user. Currently only works via `sudo`.
 
 ### `winrm` transport configuration options
 
-`connect-timeout`: Maximum amount of time to allow for a WinRM connection to be established, in seconds
+`connect-timeout`: Maximum amount of time to allow for a WinRM connection to be established, in seconds.
+
+`insecure`: Whether to skip requiring SSL for connections. (default: false)
+
+`cacert`: The CA certificate used to authenticate SSL connections. (default: uses system CA certificates)
+
+`tmpdir`: The directory to store temporary files on the target node. (default: `[System.IO.Path]::GetTempPath()`)
+
+`extensions`: List of file extensions that will be accepted for scripts or tasks. `.ps1`, `.rb`, and `.pp` are always allowed.
```

Comment:
Can you add a small note / example about how additional extensions are evaluated?  I think it would be helpful to Windows users to understand precisely what this means...

Reply:
Added.